script/__snapshots__/migrate-test-e2e.js.snap

```diff
@@ -186,7 +186,7 @@ exports[`expected file changes > README.md 1`] = `
  ## Development
  
  See [\`.github/CONTRIBUTING.md\`](./.github/CONTRIBUTING.md), then [\`.github/DEVELOPMENT.md\`](./.github/DEVELOPMENT.md).
-@@ -128,3 +91,7 @@ Thanks! 💖
+@@ -129,3 +92,7 @@ Thanks! 💖
```

Comment:
Coming over from https://github.com/JoshuaKGoldberg/create-typescript-app/pull/1148#discussion_r1438581492: this is wrong... 🤔